imap_processing/ena_maps/ena_maps.py

```diff
@@ -1155,6 +1226,43 @@ def __init__(
             }
         )
 
+    @property
+    def binning_grid_shape(self) -> tuple[int]:
+        """
+        Binning_grid_shape for consistency with RectangularSkyMap.
+
+        Returns
+        -------
+        binning_grid_shape : tuple[int]
+            Shape of the AzElSkyGrid (num_az_bins, num_el_bins).
```

Comment:
I think this is under the Healpix version now and needs the docstring updated.
```suggestion
            Shape of the HealpixSkyGrid (num_points,).
```